modules/unsupported/process-feature/src/test/java/org/geotools/process/vector/PointStackerProcessTest.java

```diff
@@ -71,14 +71,32 @@ public void testSimple() throws ProcessException, TransformException {
         PointStackerProcess psp = new PointStackerProcess();
         SimpleFeatureCollection result =
                 psp.execute(
-                        fc, 100, // cellSize
-                        null, // weightClusterPosition
-                        null, // normalize
-                        null, // preserve location
+                        fc,
```

Comment:
Looks like the test has just been adapted to follow the new interface. For the PR to be merged, you need a new test for each method and option you added in the process.

Reply:
It will be taken care of.